src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs

```diff
@@ -770,5 +778,33 @@ private void HandleCustomHostLaunch(ITestHostLauncher customHostLauncher, Messag
                 this.communicationManager.SendMessage(MessageType.CustomTestHostLaunchCallback, ackPayload, this.protocolVersion);
             }
         }
+
+        private void AttachDebuggerToProcess(ITestHostLauncher customHostLauncher, Message message)
+        {
+            var ackPayload = new EditorAttachDebuggerAckPayload() { Attached = false, ErrorMessage = null };
+
+            try
+            {
+                var pid = this.dataSerializer.DeserializePayload<int>(message);
+
+                ackPayload.Attached = customHostLauncher is ITestHostLauncher2 launcher
+                    ? launcher.AttachDebuggerToProcess(pid)
+                    : false;
```

Comment:
>false [](start = 22, length = 5)

Should we be adding some information in the Error Message here indicating why we aren't attaching?

Reply:
Failure to attach should be interpreted by TP. There might be cases where the failure is expected to happen and so for those cases we shouldn't be providing an error message. 

I'd also rather keep consistent with `HandleCustomHostLaunch` that only uses the ErrorMessage field if there's an exception that occurred during attach.